src/entities/Updates/routes.ts

```diff
@@ -117,10 +117,12 @@ async function updateProposalUpdate(req: WithAuth<Request<{ proposal: string }>>
     throw new RequestError(`Update not found: "${id}"`, RequestError.NotFound)
   }
 
-  if (update?.completion_date) {
-    throw new RequestError(`Update already completed: "${update.id}"`, RequestError.BadRequest)
+  if (!update?.completion_date) {
```

Comment:
This function is being used to update pending proposal updates. Pending updates don't have completion_date.
If I understand correctly, if you make this change, users with tier 3-6 grants can't post updates.